tests/Service/FormatSponsorshipLevelBenefitTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: btaralle
+ * Date: 2019-02-07
+ * Time: 17:41
+ */
+
+namespace App\Tests\Service;
+
+use App\Repository\SponsorshipBenefit\SponsorshipBenefitManagerInterface;
+use App\Repository\SponsorshipLevel\SponsorshipLevelManagerInterface;
+use App\Repository\SponsorshipLevelBenefit\SponsorshipLevelBenefitManagerInterface;
+use App\Service\FormatSponsorshipLevelBenefit;
+use Prophecy\Argument;
+use PHPUnit\Framework\TestCase;
+
+class FormatSponsorshipLevelBenefitTest extends TestCase
+{
+    private $sponsorshipBenefitManagerProphecy;
+    private $sponsorshipLevelManagerProphecy;
+    private $sponsorshipLevelBenefitManagerProphecy;
+    private $formatSponsorshipLevelBenefit;
+
+    public function setUp()
+    {
+        $this->sponsorshipBenefitManagerProphecy = $this->prophesize(SponsorshipBenefitManagerInterface::class);
+        $this->sponsorshipLevelManagerProphecy = $this->prophesize(SponsorshipLevelManagerInterface::class);
+        $this->sponsorshipLevelBenefitManagerProphecy = $this->prophesize(SponsorshipLevelBenefitManagerInterface::class);
+
+        $this->formatSponsorshipLevelBenefit = new FormatSponsorshipLevelBenefit(
+            $this->sponsorshipBenefitManagerProphecy->reveal(),
+            $this->sponsorshipLevelManagerProphecy->reveal(),
+            $this->sponsorshipLevelBenefitManagerProphecy->reveal()
+        );
+    }
+
+    public function testFormat()
+    {
+        $this->sponsorshipLevelManagerProphecy->getOrderedList()->shouldBeCalled()->willReturn([]);
+        $this->sponsorshipBenefitManagerProphecy->getOrderedList()->shouldBeCalled()->willReturn([]);
+
+        $this->sponsorshipLevelBenefitManagerProphecy->getByBenefitAndLevel(Argument::any(), Argument::any())->shouldNotBeCalled();
+
+        $this->formatSponsorshipLevelBenefit->format();
```

Comment:
Am I missing something here, or is the assertion missing?

Reply:
resolve with https://github.com/DarkmiraTour/community-event-manager/pull/92/commits/c8bcbae1d898e58447578c40a3dbad4c8abdbf02